packages/d3fc-webgl/src/series/glArea.js

```diff
@@ -32,9 +33,8 @@ export default () => {
             [0, 0, 1],
             [1, 0, 0],
             [1, 1, 0]
-        ])
-        .value((data, element, vertex, component) => data[vertex][component]);
-    const definedAttribute = projectedAttributeBuilder().value(
+        ]);
+    const definedAttribute = elementConstantAttributeBuilder().value(
```

Comment:
Should this definitely be elementConstantAttribute builder here? It appears to work.

Reply:
Yep as discussed.